translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) | [:es: Español](README.spanish.md)          ￨
```

Comment:
This doesn't show up correctly yet:

![image](https://user-images.githubusercontent.com/5722022/67633060-70f96780-f8ab-11e9-8cad-62dcbc437bbc.png)

Reply:
Done mam